expression-language.md

```diff
@@ -361,14 +351,7 @@ operators.
 
 #### 3.7. Type casting
 
-CESQL supports both implicit and explicit type casting among the _primitive_ types. Users can perform explicit type
-casting through the functions defined in the [Casting and type checking](#351-casting-and-type-checking) sub-paragraph.
-
-When input parameters' types of operator/function don't match the signatures, the CESQL engine MUST try to perform an
-implicit cast.
-
-Implicit casts must follow the same semantics of their equivalent explicit cast functions, as defined in
-[Casting and type checking](#351-casting-and-type-checking) sub-paragraph.
+When input parameters' types of operator/function don't match the signatures, the CESQL engine MUST try to perform an implicit cast.
```

Comment:
It's possible that it's just because I'm not really familiar with CESQL, but I find this sentence hard to read. I suspect that making it longer may help.

(Additionally, we may wish to consider using "argument" and "parameter" terminology in the same way as it's used in various programming languages, where the parameter is what gets declared by the function/operator/whatever, and the argument is what is passed in by the caller. I haven't checked whether that's been used elsewhere already. If folks think it would be valuable for me to look over the CESQL spec and *maybe* make a bigger proposal on this, I'd be happy to do so. No promises when!)

Reply:
I did not add the sentence, it is just a diff glitch, the PR is about removing reference to explicit cast, but I'm fine with a review and update of the spec like you proposed, probably in another PR.